javaScript/featureSection.js

```diff
@@ -51,7 +51,7 @@ const makeGames = () => {
     const gameHtml = `
       <div class='feature-section__game-container cont${i}'>
         <div class='feature-section__image-container'>
-        <img class='feature-section__image' src='${currentGame.image}' alt='game image'>
+        <img class='feature-section__image' src='${currentGame.image}' alt='${currentGame.title} image'>
         <img class='feature-section__chess' src='images/chessBoard.png' alt='background image'>
         </div>
         <ul class='feature-section__list'>
```

Comment:
Great design @EvansSnave 🤝

- [ ] [Optional] I would recommend your design matches the given design as much as possible.
- Your design
![Your features](https://github.com/EvansSnave/capstone-project/assets/63917725/ac99ed01-a3fa-40ef-b958-496bd0640f40)
- Given design
![Given-features](https://github.com/EvansSnave/capstone-project/assets/63917725/781d9259-4184-4c1d-912c-cb4993456da5)